src/modules/whats-new-modal/whats-new-content.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import ButtonComponent from 'src/components/button';
+
+interface Link {
+	url?: string;
+	text: string;
+	icon?: string;
+}
+
+interface WhatsNewContentProps {
+	title: string;
+	description: React.ReactNode;
+	links?: Link[];
+	illustration?: string;
+	illustrationComponent?: React.ComponentType;
+	className?: string;
+	width?: string | number;
+	height?: string | number;
```

Comment:
Generally (once we have the design), I think we could aim for keeping all the content items the same size.

Looking at the existing WordPress.com "What's New" modal, there's enough space provided in case the translations are longer in different locales:

![Markup on 2025-03-21 at 11:51:42](https://github.com/user-attachments/assets/f5dc4c5d-26ca-4219-90e1-5acf2e2f9c5c)